src/components/CreateModal/CreateModal.js

```diff
@@ -54,36 +54,41 @@ export default function CreateModal (props) {
             <Icon name='Ex' />
           </span>
           { postId && action === 'edit'
-            ? <PostEditor
-              {...props}
-              onClose={closeModal}
-              setIsDirty={setIsDirty}
-            />
-            : <Switch>
-              <Route
-                path={match.path + '/post'}
-                children={({ match, location }) => (
-                  <PostEditor
-                    {...props}
-                    onClose={closeModal}
-                    setIsDirty={setIsDirty}
-                  />
-                )}
+            ? (
+              <PostEditor
+                {...props}
+                onClose={closeModal}
+                onCancel={confirmClose}
```

Comment:
this stuff might have some conflicts with the changes I made to this code on dev?

Reply:
This branch was merged with the current dev right up until Tom merged Group Explorer this afternoon so all of that was reconciled and tested. As a rule I merge up from dev and test before reviews.